packages/ccdi-cde/src/v1/sample/library_source_material.rs

```diff
@@ -0,0 +1,165 @@
+    ///
+    /// * **VM Long Name**: Bulk Nucleus Specimen
+    /// * **VM Public ID**: 7592129
+    /// * **Concept Code**: C178224
+    /// * **Begin Date**:   02/28/2024
+    ///
+    /// A biospecimen consisting of multiple nuclei intended to be analyzed as a pool.
+    #[serde(rename = "Bulk Nuclei")]
+    BulkNuclei,
+
+    /// `Bulk Tissue`
+    ///
+    /// * **VM Long Name**: Bulk Tissue Specimen
+    /// * **VM Public ID**: 7592128
+    /// * **Concept Code**: C178225
+    /// * **Begin Date**:   02/28/2024
+    ///
+    /// A biospecimen either derived from a whole tissue specimen or tissue section, which may consist of heterogeneous cells or tissues.
+    #[serde(rename = "Bulk Tissue")]
+    BulkTissue,
+
+    /// `Single-cells`
+    ///
+    /// * **VM Long Name**: Single Cell Suspension
+    /// * **VM Public ID**: 14838800
+    /// * **Concept Code**: C204464
+    /// * **Begin Date**:   03/12/2024
+    ///
+    /// A dilute suspension of cells intended to be further fractionated for assays focused on single-cells.
+    #[serde(rename = "Single-cells")]
+    SingleCells,
+
+    /// `Single-nuclei`
+    ///
+    /// * **VM Long Name**: Single Nucleus Suspension
+    /// * **VM Public ID**: 14838802
+    /// * **Concept Code**: C204465
+    /// * **Begin Date**:   03/12/2024
+    ///
+    /// A dilute suspension comprised of isolated intact cell nuclei intended to be further fractionated for assays focused on single-nuclei.
+    #[serde(rename = "Single-nuclei")]
+    SingleNuclei,
+
+    /// `Not Reported`
+    ///
+    /// * **VM Long Name**: Not Reported
+    /// * **VM Public ID**: 5612322
+    /// * **Concept Code**: C43234
+    /// * **Begin Date**:   03/01/2024
+    ///
+    /// Not provided or available.
+    #[serde(rename = "Not Reported")]
+    NotReported,
+}
+
+impl CDE for LibrarySourceMaterial {}
+
+impl std::fmt::Display for LibrarySourceMaterial {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            LibrarySourceMaterial::BulkCells => write!(f, "Bulk Cells"),
+            LibrarySourceMaterial::BulkNuclei => write!(f, "Bulk Nuclei"),
+            LibrarySourceMaterial::BulkTissue => write!(f, "Bulk Tissue"),
+            LibrarySourceMaterial::SingleCells => write!(f, "Single-cells"),
+            LibrarySourceMaterial::SingleNuclei => write!(f, "Single-nuclei"),
+            LibrarySourceMaterial::NotReported => write!(f, "Not Reported"),
+        }
+    }
+}
+
+impl Distribution<LibrarySourceMaterial> for Standard {
+    fn sample<R: rand::Rng + ?Sized>(&self, rng: &mut R) -> LibrarySourceMaterial {
+        match rng.gen_range(0..5) {
+            0 => LibrarySourceMaterial::BulkCells,
+            1 => LibrarySourceMaterial::BulkNuclei,
+            2 => LibrarySourceMaterial::BulkTissue,
+            3 => LibrarySourceMaterial::SingleCells,
+            4 => LibrarySourceMaterial::SingleNuclei,
+            _ => LibrarySourceMaterial::NotReported,
```

Comment:
@e-t-k  @claymcleod  Is this ok to leave this as catchall since we don't have Other anymore?

Reply:
Yup! Looks good.